redis/rediscache_test.go

```diff
@@ -0,0 +1,45 @@
+package rediscache
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/soveran/redisurl"
+)
+
+const testServer = "redis://localhost:6379"
+
+func TestRedisCache(t *testing.T) {
+	conn, err := redisurl.ConnectToURL(testServer)
```

Comment:
This [`github.com/soveran/redisurl`](https://godoc.org/github.com/soveran/redisurl) dependency seems unnecessary given the simple `testServer` address. I think it can be replaced with just:

```Go
conn, err := redis.Dial("tcp", "localhost:6379")
```